news/2017-08-30-qat-gazette-august-2017.md

```diff
@@ -0,0 +1,41 @@
+
+## Updates to the Beatmap Nomination Group
+
+Firstly, **please welcome two new Beatmap Nominators to the Advanced Tier for osu!standard - Battle and Naxess!** As you may be aware, Advanced Tier Nominators are responsible for beatmap qualification, and with the recent departure of a number of existing members, it’s important that receiving a heart icon doesn’t become a bottleneck for mappers. Please treat them well as they settle into their new role! On the other side of the coin, **a number of existing members of the Beatmap Nomination Group will be departing** to pursue other opportunities. Many thanks to Dellvangel, -Kamikaze-, Kuo Kyoka, newyams99, Nivrad00 and Sonnyc for your contributions to our community over the months and years!
+
+Fear not however, as **we are currently reviewing applications for a new batch of fresh-faced modders to replenish their ranks!** The latest application process for the osu!taiko, osu!catch and osu!mania! game modes has recently closed, with successful applicants expected to promoted in the coming weeks. With the discontinuation of Beatmap Nominator tests, new BN applications will consist of an activity check, as well as submission of the applicant’s “best” modding work - either as a standard mod post, or series of suggestions in the moddingv2 panel. For those that meet the activity requirements, the QAT will then supplement this with additional random, anonymised examples of the applicant’s modding work from the past three months and evaluate these examples on a number of criteria, such as structure, helpfulness, knowledge of the ranking criteria, and so on. These modding examples will be voted on by the remaining members of the QAT who were not involved in selecting examples of applicant modding, to avoid bias. **Those with a majority of positive votes will be promoted into one of the available** [Nominator Subdivisions](https://osu.ppy.sh/forum/p/6201585) and closely surveyed on their general attitude (forum posts, in-game chat, modding and icon activity, etc.) to ensure they receive the assistance needed to fulfill their roles as gatekeepers of content in the mapping and modding community.
```

Comment:
new paragraph at coming weeks

new paragraph at 'for those who meet the activity requirements'

new paragraph at 'those with a majority of positive votes'

Reply:
adding paragraphs faster than I kill maps in the inappropriate beatmap submissions forum